sites/platform/src/domains/cdn/_index.md

```diff
@@ -62,6 +62,8 @@ see your CDN provider's official documentation.
 When you use a CDN, the {{< vendor/name >}} router [HTTP caching](../../define-routes/cache.md) becomes redundant.
 To disable it, change your cache configuration for the routes behind a CDN to the following:
 
+{{< version/specific >}}
```

Comment:
@chadwcarlson This works and I wanted to replicate that on l.154-164 to show the `routes` top-level key for Upsun in the conf. But the shortcode isn't rendering well (because it's in an ordered list?) and I couldn't fix it.